english/zuluquest/newpath.md

```diff
@@ -0,0 +1,10 @@
+OK! So let's give you some extra background here...
+
+Let's say you worked for more than 10 years doing software for banks, insurance/pharmaceutical/motor companies and public sector.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.